### ISSUE TITLE
Let us hide the bounds of GUI nodes

### ISSUE THREAD

**Issue by subsoap** [defold/editor2-issues#2909]
We can hide the GUI bounds of the scene, but not the bounds of nodes themselves. 

Would be useful to hide node bounds to be able to cleanly see layouts.

**Comment by Jerakin** [comment]
You can do this with View -> Toggle Component Guides

**Comment by subsoap** [comment]
@Jerakin Thanks! Strange that it's not on the visibility filter list.

**Comment by Jerakin** [comment]
I agree, maybe change this/create new issue for adding it to the visibility filters?
